### DISCUSSION TITLE
Send Attachment through Automate

### DISCUSSION THREAD

**Discussion by timchkor** [Budibase/budibase#1689] [Ideas]
Hello, 

It would be great if it's possible to send Attachment stored in Data table through 'Automate' - 'Send SMTP Email' and 'Outgoing Webhook'.

![image](https://user-images.githubusercontent.com/85532850/121471640-0010b000-c9fb-11eb-9e35-943363e7938b.png)

Thx!

**Comment by mjashanks** [comment]
Thanks! Not possible right now, but changed this to an Idea.

Out of interest, would it be enough to provide a link to the file in the email, so the user could download - or does it need to be an attachment.

Just checking - makes sense either way.

**Reply by agaheu** [reply]
Hi Michael,
Is there a ready to use way providing links for attachments?

**Reply by mjashanks** [reply]
Hey @agaheu 

Yes - steps below will list all attachments in a table
1. Add a `Repeater Block`, connected to your table. (A repeater block is the same as a `Data Provider` + `Repeater`)
2. Add a second `Repeater Block`, this time connect the data to `Fields > Your Attachment Column`
3. Add elements in here to display details of the attachment. In my case, I am just displaying the URL in a paragraph :
`{{ Attachments repeater.Docs.url }}`

Video of my setup:

https://user-images.githubusercontent.com/3524181/157256957-467a3ae4-7223-4f1b-aee6-bf9c9211c3bc.mp4

**Comment by dobby8930** [comment]
For me attachments is better

**Comment by giovannytj10** [comment]
Is there any way that when a ticket is created in which an email is added, that email is linked to the one that will be sent in the automation?

**Reply by Nasnl** [reply]
What I do is to make it a field (I call it {{sender }}), and then pass it on through binding and use that as the 'To:' field.

Hope this helps.
